x-pack/plugins/enterprise_search/server/__mocks__/routerDependencies.mock.ts

```diff
@@ -7,7 +7,10 @@
 
 import { loggingSystemMock } from 'src/core/server/mocks';
 
-import { ConfigType } from '../';
+import {
+  EnterpriseSearchConfig,
+  EnteepriseSearchConfigType,
```

Comment:
typo in the name here - looks like this should be 
```suggestion
  EnterpriseSearchConfigType,
```